molecule/commands/syntax.py

```diff
@@ -44,8 +44,7 @@ def execute(self, exit=True):
         ansible = ansible_playbook.AnsiblePlaybook(self.molecule.config.config[
             'ansible'])
         ansible.add_cli_arg('syntax-check', True)
-        ansible.add_cli_arg('inventory-file', 'localhost,')
-
+        ansible.add_cli_arg('inventory_file', 'localhost')
```

Comment:
This is actually the bug fix.  I prob should have broken this out into two commits.